src/app/ASKS/ask/index.js

```diff
@@ -671,8 +688,8 @@ const Ask = ({
           clientId={clientId}
         />
       )}
-      {component === 'checkbox' && (
-        <CheckBox.Write
+      {component === 'single_checkbox' && (
```

Comment:
Instead of having two separate component name, **single_checkbox** and **checkbox**, let's just have a single component name of **checkbox**.
We can then further distinguish between them using the type keyword. The type of our existing checkbox which let's user toggle between single value can be **single_checkbox** where as the checkbox that lets user select from range of options can be the default one. 
We will then render single check box component if the type is **single_checkbox**, the default one otherwise.